setup.cfg

```diff
@@ -21,6 +21,9 @@ setup_requires =
     setuptools_scm
 install_requires =
     jsonschema>=3.0.2
+    # The tests require code that is in asdf master
+    # but not yet released.  The package itself
+    # is compatible with asdf 2.7.
```

Comment:
I tried adding an additional `asdf @ git ...` dependency in the `[test]` extras and it had no effect.  I guess pip figures it's doing us a favor by not installing it twice.

Reply:
Right.  It must be because it can't resolve versions because when installing from Github, there's no version specifier, and it needs to be able to resolve it before it installs it.